### PR TITLE
ManageStudentsTable refactors for Dev Book Club!

### PR DESCRIPTION
**Pull request by maureensturgeon**
@made-line and I applied the concepts in Clean Code Chapter 3: Functions to refactor some of the methods in the ManageStudentsTable component!

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by clareconstantine — APPROVED** [review]
This is so much cleaner!

File: apps/src/templates/manageStudents/ManageStudentsTable.jsx [hunk 1]
```diff
@@ -426,156 +431,173 @@ class ManageStudentsTable extends Component {
 
   getColumns(sortable) {
     const {loginType} = this.props;
-    let dataColumns = [
```
Comment (Erin007) [inline apps/src/templates/manageStudents/ManageStudentsTable.jsx:429]: I think you'll see this pattern all over LP land. Would be nice to sneak in a refactor of other tables as an incremental follow-up.

**Review by Erin007 — APPROVED** [review]
YAY! So much tidier! Thanks for taking the initiative to clean this up!

**Comment by Erin007** [comment]
tidy

File: apps/src/templates/manageStudents/ManageStudentsTable.jsx [hunk 2]
```diff
@@ -426,156 +431,173 @@ class ManageStudentsTable extends Component {
...
+      },
+      cell: {
+        formatters: [this.actionsFormatter],
+        props: {
+          style: {
+            ...tableLayoutStyles.cell
+          }
+        }
+      }
+    };
```
Comment (davidsbailey) [inline apps/src/templates/manageStudents/ManageStudentsTable.jsx:600]: nice extractions!

**Pull request merged**